requirements.txt

```diff
@@ -81,8 +81,9 @@ parsimonious==0.6.2 \
 polib==1.0.6 \
     --hash=sha256:b1ea141d58ed5e48aed2674f7c894dfb83f639c3286d7b32b2e19fa032a5b400 \
     --hash=sha256:20d2a0d589a692c11df549bd7cda83c665eef2a83e017b843fecdf956edbad74
-psycopg2==2.6 \
-    --hash=sha256:c00afecb302a99a4f83dec9b055c4d1cc196926d62c8db015d68432df8118ca8
+psycopg2==2.7.3.2 \
```

Comment:
@mathjazz 
I don't see anything that could break our code in psycopg2: http://initd.org/psycopg/docs/news.html
Even people from django suggest to use newest version: https://docs.djangoproject.com/en/1.10/ref/databases/#postgresql-notes (sorry, I couldn't find a link to docs for 1.10.8).